utils/metrics.py

```diff
@@ -1,17 +1,25 @@
+from ast import Raise
+from xml.dom import ValidationErr
 import numpy as np
 from sklearn.metrics import classification_report
+from math import sqrt
+from torch import IntTensor
+from torchmetrics import JaccardIndex
 
 min_val = 1e-6
-def create_metrics_dict(num_classes):
-    num_classes = num_classes if num_classes == 1 else num_classes + 1
+def create_metrics_dict(num_classes, ignore_index=None):
+
+    num_classes = num_classes + 1 if num_classes == 1 else num_classes
```

Comment:
That's a pretty darn subtle bug :(
High five to the digital accelerator folks...

Reply:
Yup. But with the tests, we'll see these errors now!